### DISCUSSION TITLE
Poll: shall Sam Astro continue this amazing project.

### DISCUSSION THREAD

**Discussion by turrnut** [sam-astro/Z-Sharp#48] [Polls]
I just saw in his README file he said he no longer maintaining this project. But personnally I think this is a great project and want him to continue, or shall he?

**Comment by sam-astro** [comment]
Lol. I don't mind if anybody else works on it or adds features, but I'm done with it haha

**Reply by turrnut** [reply]
ok

**Comment by HamzaVextor** [comment]
Hey im new to github can i ask how can i contribute in Zsharp

**Reply by sam-astro** [reply]
You can **make a fork** of it (https://docs.github.com/en/get-started/quickstart/fork-a-repo)
Then make your changes to that fork.
When you are done making changes and want me to add them to the main repository, make a **pull request**

**Reply by turrnut** [reply]
things are getting more interesting, **sus**

**Reply by sam-astro** [reply]
Haha sorry, but I don't think I want to sell this project lol. It wouldn't even be worth anything anyway, since it is programmed terribly. 😅

**Reply by sam-astro** [reply]
@Hamza1411  If you want just contact me on discord: Samm#0203

**Reply by turrnut** [reply]
@sam-astro Hi can I add your discord(my discord is turrnut#9727), I think you are just talented to create a programming language and would like to know a few more kind of people. Can you add me? Thanks!

**Reply by sam-astro** [reply]
You can join my server at this link: [https://discord.gg/9p82dTEdkN](https://discord.gg/9p82dTEdkN)
